.iex.exs

```diff
@@ -33,3 +34,7 @@ alias Aeutil.Scientific
 alias Aeutil.Serialization
 alias Aeutil.PatriciaMerkleTree
 alias Aecore.Chain.Identifier
+
+alias Aevm.State
+
+require Aecore.Contract.ContractConstants, as: Constants
```

Comment:
this should be ContractConstants

Reply:
why is this require not alias?

Reply:
you need to require the module to use its macros

Reply:
:+1: